packages/cactus-plugin-ledger-connector-ethereum/package.json

```diff
@@ -65,11 +65,13 @@
     "webpack:dev:web": "webpack --env=dev --target=web --config ../../webpack.config.js"
   },
   "dependencies": {
+    "@apidevtools/json-schema-ref-parser": "^12.0.1",
```

Comment:
@Harshdev098 Please pin to exact versions instead of having auto-upgrade enabled. It's not a surefire way to protect against supply-chain attacks but better than nothing.